queries.sql

```diff
@@ -79,7 +79,47 @@ GROUP BY
     species;
 
 
+SELECT animals.name
+FROM animals
+JOIN owners
+ON animals.owners_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.species_name = 'Pokemon';
 
 
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owners_id;
 
 
+SELECT species.species_name, COUNT(animals.id) AS animal_count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.species_name;
```

Comment:
- [ ] The screenshot attached for this is not quite correct because from this milestone requirements, we can count number of `digimons` to be 4 (`Pikachu`, `Charmander`, `Squirtle`, and `Blossom`) before taking a look into previous data from previous milestones which one might find out about `Ditto`. I think you may have to check this :pray: 
![image](https://github.com/Xghayor/vet-clinic-db/assets/99715640/1e353554-a38e-418b-9950-b45c2dfad746)